test/Dialect/Calyx/emit-custom-primitives.mlir

```diff
@@ -0,0 +1,40 @@
+// RUN: circt-translate --export-calyx --verify-diagnostics %s | FileCheck %s --strict-whitespace
```

Comment:
Can you add a test for this in https://github.com/llvm/circt/blob/main/test/Dialect/Calyx/round-trip.mlir as well please?